src/views/Room.vue

```diff
@@ -93,12 +93,14 @@ export default {
 
       rtc.on("signaling_not_reachable", () => {
         logger.error("signaling server not reachable")
-        this.reconnectRtc()
+        // this.reconnectRtcWhenOnLine()
```

Comment:
...or not? I'm not so sure anymore. signaling_not_reachable is thrown when either
1) the socket state goes to closing or closed
2) the socket failed connecting in the first place

In case 1, the reconnect could actually be helpful. I guess we should sort out in palava-client when to throw signaling_not_reachable and when signaling_error, because signaling_not_reachable already combines two very different things...

Reply:
Just saw that nr. 2 is not the case anymore ;-) thus: I think signaling_not_reachable should be a reason for a reconnect (because this can happen at any time even if the connection was established before, right?)

Reply:
Did you see this? The rest looks good now!

Reply:
We should already catch connection errors by all other means. However, since `reconnectRtcWhenOnLine()` will only register the online handler once, we could also just add it. Or leave it out for now (and only log it) and see how it goes?

Reply:
let's add it to be sure? I mean it doesn't hurt I think